pkgs/tools/backup/gphotos-sync/default.nix

```diff
@@ -1,54 +1,81 @@
 { lib
 , fetchFromGitHub
+, fetchpatch
+, pre-commit
 , python3Packages
 , ffmpeg
 }:
-
+let
+  patched-google-auth-oauthlib =
+    python3Packages.google-auth-oauthlib.overrideAttrs (_: {
+      patches = [
+        (fetchpatch {
```

Comment:
Override `version = "0.5.2b1"` instead.

Reply:
using the fork as `src` seems more correct to me since that's what upstream does and is less likely to break

Reply:
I was thinking about the case where `google-auth-oauthlib` was updated (say due to a a security issue) and we wanted to get that update asap. But overall I agree `src` is less likely to break, so I've changed it.